src/block/tile/Sign.php

```diff
@@ -77,22 +77,29 @@ public function __construct(World $world, Vector3 $pos){
 		parent::__construct($world, $pos);
 	}
 
+	private function readTextTag(CompoundTag $nbt) : void{
+		$baseColor = new Color(0, 0, 0);
+		$glowingText = false;
+		if(($baseColorTag = $nbt->getTag(self::TAG_TEXT_COLOR)) instanceof IntTag){
+			$baseColor = Color::fromARGB(Binary::unsignInt($baseColorTag->getValue()));
+		}
+		if(
+			($glowingTextTag = $nbt->getTag(self::TAG_GLOWING_TEXT)) instanceof ByteTag &&
+			($lightingBugResolvedTag = $nbt->getTag(self::TAG_LEGACY_BUG_RESOLVE)) instanceof ByteTag
```

Comment:
This tag is no longer present with the FrontText/BackText change, which means that all glowing signs stored in the new format won't be handled correctly by this change.